packages/@expo/cli/src/start/doctor/dependencies/ensureDependenciesAsync.ts

```diff
@@ -77,7 +75,12 @@ export async function ensureDependenciesAsync(
     title = '';
   }
 
-  const installCommand = createInstallCommand({ isYarn, packages: missing });
+  const managerName = PackageManager.resolvePackageManager(projectRoot);
```

Comment:
Just a random thought, but would it help to have a "default" manager returned from this too, or as constant?